### PR TITLE
Add Turkish motorway shields

### PR DESCRIPTION
**Pull request by claysmalley**
Adds shield support for motorways of Turkey.

Screenshot from 2022-07-13 18-59-44
O3-TR_sign

### CODE REVIEW

File: src/js/shield_defs.js [hunk 1]
```diff
@@ -2716,6 +2704,26 @@ export function loadShields(shieldImages) {
...
+    Color.shields.white,
+    Color.shields.black
+  );
+
+  // Philippines
+  shields["PH:N"] = homeDownShield;
+  shields["PH:E"] = homeDownShieldYellow;
+
+  // Turkey
+  shields["TR:motorway"] = hexagonVerticalShieldOrange;
```
Comment (1ec5) [inline src/js/shield_defs.js:2725]: Nit: “Taiwan” sorts before “Turkey” (or “Türkiye”).

**Review by 1ec5 — APPROVED** [review]
Interesting that the dash looks so close to an underscore or period on the actual signs, but I suppose that’s just a typographical flourish.

Let’s ticket out shields for Turkey’s state roads (D) and, if signposted, the provincial roads too.

File: src/js/shield_defs.js (hunk 1 above)
Comment (claysmalley) [reply src/js/shield_defs.js:2725]: That's intentional; it's sorted by ISO country code.

**Comment by claysmalley** [comment]
State roads tracked here: #494

**Pull request merged**